CMakeLists.txt

```diff
@@ -291,14 +254,13 @@ function(create_library)
     target_link_libraries(${PROJECT_NAME} PUBLIC OpenMP::OpenMP_CXX)
   endif()
 
-  target_link_libraries(${PROJECT_NAME} PUBLIC proxsuite-nlp::proxsuite-nlp)
-  target_link_libraries(${PROJECT_NAME} PUBLIC Boost::boost)
   target_link_libraries(${PROJECT_NAME} PUBLIC fmt::fmt)
-  # set the install-tree include dirs
-  # used by dependent projects to consume this target
+  target_link_libraries(${PROJECT_NAME} PRIVATE Tracy::TracyClient)
+  # set include dirs used by dependent projects to consume this target
   target_include_directories(
     ${PROJECT_NAME}
-    PUBLIC $<INSTALL_INTERFACE:include> $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/include>
+    PUBLIC $<INSTALL_INTERFACE:include> $<INSTALL_INTERFACE:${THIRD_PARTY_HEADERS_INSTALL_DIR}>
```

Comment:
Hm. What should we do then? Since we still want to do `#include <tracy/Tracy.hpp>` ?

Reply:
I don't know…
Since we take tracy headers from submodule to build we can't easily add a prefix to the tracy include directory.
Best option is to copy tracy headers in a custom build directory.

Reply:
Copy or symlink them to the build-tree? That could work.

Reply:
I prefer copy because of Windows :p

Reply:
Okay so I copied over all the headers from `tracy/client` and `tracy/common` and the only relevant one from `tracy/tracy`